bigframes/ml/llm.py

```diff
@@ -113,6 +115,23 @@ def _create_bqml_model(self):
             session=self.session, connection_name=self.connection_name, options=options
         )
 
+    @classmethod
+    def _from_bq(
+        cls, session: bigframes.Session, model: bigquery.Model
+    ) -> PaLM2TextGenerator:
+        if (
```

Comment:
The logic is already checked in loader.py, and it should be there. No need to do it again. (But you can assert the truth)

Reply:
Done.